.github/workflows/auto-merge.yml

```diff
@@ -0,0 +1,34 @@
+name: Auto Merge
+
+on:
+  pull_request:
+    branches:
+      - opendistro-*
+    types:
+      - labeled
```

Comment:
What is the behavior for PRs made to master (not to be backported/automerged) and a label is added. 
Does this workflow kick in ?